tests/e2e-pw/specs/merchant/merchant-admin-account-balance.spec.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { useMerchant } from '../../utils/helpers';
+
+test.describe(
+	'Merchant account balance overview for single deposit currency accounts',
+	() => {
+		// Use the merchant user for this test suite.
+		useMerchant();
+
+		test(
+			'View the total and available account balance for a single deposit currency (USD)',
+			{
+				tag: '@critical',
+			},
+			async ( { page } ) => {
+				await test.step(
+					'Navigate to the Payments Overview screen',
+					async () => {
+						await page.goto(
+							'/wp-admin/admin.php?page=wc-admin&path=/payments/overview'
+						);
+					}
+				);
+
+				await test.step(
+					'Observe the total account balance, ensuring it has a dollar value with the correct currency formatting',
+					async () => {
+						const totalBalanceValue = page.getByLabel(
+							'Total balance',
+							{
+								exact: true,
+							}
+						);
+
+						// Match the total balance value to the USD format $1*
+						// Intentionally not expecting a specific value to allow for different values in different environments.
+						await expect( totalBalanceValue ).toHaveText( /\$\d+/ );
+					}
+				);
+
+				await test.step(
+					'Observe the available account balance, ensuring it has a dollar value with the correct currency formatting',
+					async () => {
+						const availableFundsValue = page.getByLabel(
+							'Available funds',
+							{
+								exact: true,
+							}
+						);
+						await expect( availableFundsValue ).toHaveText(
+							/\$\d+/
+						);
+					}
+				);
+			}
+		);
+	}
+);
+
+test.describe(
+	'Merchant account balance overview for multiple deposit currency accounts',
+	{
+		tag: '@critical @todo',
```

Comment:
According to the Playwright documentation, it seems the recommended way to add multiple tags is by using an array, so this line could be something like:
```tag: ['@critical', '@todo'],```

Source: https://playwright.dev/docs/test-annotations#tag-tests

Reply:
Thanks for catching this – resolved in 685ba12